apps/trading/components/footer/footer.tsx

```diff
@@ -65,9 +65,8 @@ interface NodeUrlProps {
 }
 
 export const NodeUrl = ({ url }: NodeUrlProps) => {
-  // get base url from api url, api sub domain
   const urlObj = new URL(url);
-  const nodeUrl = urlObj.origin.replace(/^[^.]+\./g, '');
+  const nodeUrl = urlObj.origin;
```

Comment:
```suggestion
  const nodeUrl = urlObj.hostname;
```

**thought (non-blocking)**: Non blocking, but this would give (e.g. `darling.network` rather than `https://darling.network`, or `n09.d.vega.xyz` rather `https://n09.d.vega.xyz`. Both are OK though.